<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="loginFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QComboBox" name="loginAccountType">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>251</width>
       <height>26</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип записи:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="usersTable">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>251</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchField">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>251</width>
       <height>26</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Поиск</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пользователь:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginPassword">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>390</y>
       <width>251</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Войти в учётную запись</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>440</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginAsAdminButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>480</y>
       <width>181</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Войти как адимистратор</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="signup_frame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>361</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QComboBox" name="signupAccountType">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>241</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип записи:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="signupPassword">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>440</y>
       <width>251</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Создать учётную запись</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="signupStackedWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>251</width>
       <height>271</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Почта:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер телефона:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ФИО:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="employeeFullname">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="employeeEmail">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="employeePhone">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Склад:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="employeeWarehouse">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>240</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ФИО:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userFullname">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер телефона:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userPhone">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Почта:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userEmail">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Почта:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер телефона:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ФИО или название компании:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ownerName">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ownerEmail">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ownerPhone">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ИНН</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ownerTaxNumber">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="signupButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
